pkg/rpmmd/repository.go

```diff
@@ -23,6 +23,7 @@ type repository struct {
 	CheckGPG       bool     `json:"check_gpg,omitempty"`
 	IgnoreSSL      bool     `json:"ignore_ssl,omitempty"`
 	RHSM           bool     `json:"rhsm,omitempty"`
+	ModuleHotfixes bool     `json:"module_hotfixes,omitempty"`
```

Comment:
I wonder if it wouldn't make more sense and make the unmarshaling simpler if you used `*bool` instead of `bool`. Because that way, we could distinguish explicitly set `module_hotfixes` value in the repo config vs. the value not being set and being just the default value of the type. :thinking: